core/store/src/db/rocksdb.rs

```diff
@@ -459,18 +460,18 @@ impl RocksDB {
     /// unrecoverable in most cases.
     fn pre_write_check(&self) -> Result<(), PreWriteCheckErr> {
         let counter = self.check_free_space_counter.fetch_add(1, Ordering::Relaxed);
-        if self.check_free_space_interval >= counter {
+        if CHECK_FREE_SPACE_INTERVAL >= counter {
             return Ok(());
         }
         self.check_free_space_counter.swap(0, Ordering::Relaxed);
 
         let available = available_space(self.db.path())?;
 
-        if available < 16_u64 * self.free_space_threshold {
+        if available < 16_u64 * FREE_SPACE_THRESHOLD {
```

Comment:
might pull this into a constant as well? I'd maybe warn earlier than 256 megs before death.